### DISCUSSION TITLE
What's the benefit of learning GraphQL in order to manage complex collections?

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#10716] [Archive]
I'm planning to use Directus with some rather complex collections: 
- parent tables filtered on user, status, date, author with
-   each row of it related to multiple filtered lists on the parent properties and also on each item's user, status, date and author.

Is it of interest to invest learning some **GraphQL** to get shortcuts to describe, build, maintain  and integrate these complex queries within Directus, (directly or through extensions)?

**Comment by rijkvanzanten** [comment]
It's totally personal preference. The REST API can do the exact same things as the GraphQL endpoint, so if you're more comfortable with making "regular" REST requests, I'd recommend sticking with that 👍🏻
